lldb/source/Core/FormatEntity.cpp

```diff
@@ -658,6 +658,33 @@ static char ConvertValueObjectStyleToChar(
   return '\0';
 }
 
+static bool DumpValueWithLLVMFormat(Stream &s, llvm::StringRef options,
+                                    ValueObject &target) {
```

Comment:
Can we named this something other that `target`? `target` usually means a lldb_private::Target in lldb code. Maybe `valobj`